Obsidian/WorldData/World.cs

```diff
@@ -455,6 +464,26 @@ public async Task ScheduleBlockUpdateAsync(BlockUpdate blockUpdate)
 
     public async Task ManageChunksAsync()
     {
+        // Check for chunks to unload every 30 seconds
+        if (LevelData.Time > 0 && LevelData.Time % (20 * 30) == 0)
+        {
+            List<(int X, int Z)> chunksToKeep = new();
+            this.Players.Where(p => p.Value.World == this).ForEach(p =>
+            {
+                chunksToKeep.AddRange(p.Value.client.LoadedChunks);
+            });
+
+            LoadedChunks.Except(chunksToKeep).Except(SpawnChunks).ForEach(async c =>
+            {
+                if (LoadedChunks.TryRemove(c))
+                {
+                    var r = GetRegionForChunk(c.X, c.Z);
+                    await r.UnloadChunk(c.X, c.Z);
+                }
+
+            });
```

Comment:
This is probably going to be really slow:
1. LINQ
2. many temporary allocations
3. file IO

Maybe add a TODO comment to optimize this? (Write out iterations instead of LINQ, avoid allocating where unnecessary, `await Task.WhenAll(...);` for IO...)

Reply:
I dunno... what's the point of having linq if you can't use it. It reads so well.